src/cli/migrate/index.ts

```diff
@@ -0,0 +1,54 @@
+import { log } from '../../utils/logging';
+
+import { CURRENT_NODE_LTS, nodeVersionMigration } from './nodeVersion';
+
+const migrations = {
```

Comment:
Would it be worth adding a type for this to ensure that other migrations added conform to a specific shape?